darwin/importer/formats/csvtagsvideo.py

```diff
@@ -25,12 +25,16 @@ def parse_file(path: Path) -> Optional[List[dt.AnnotationFile]]:
             start_frame = int(start_frame)
             end_frame = int(end_frame)
 
-            annotation = dt.make_video([dt.make_keyframe(dt.make_tag(tag), start_frame)], start_frame, end_frame)
+            annotation = dt.make_tag(tag)
+            frames = {i: annotation for i in range(start_frame, end_frame + 1)}
+            keyframes = {i: i == start_frame for i in range(start_frame, end_frame + 1)}
```

Comment:
Are we sure about the `+ 1` here?

Reply:
I'm not sure about that. @simedw ?

Reply:
yeah, otherwise it's not inclusive:
```py
>>> list(range(1,3))
[1, 2]
```

Reply:
What I didn't know is if the end frame indicated in the csvtag is included or not. It it like this `[start, end]` or like this `[start, end)`?